Closure_Project/rest_api/models.py

```diff
@@ -101,6 +103,21 @@ class Meta:
     def __str__(self):
         return f'{self.track_number} {self.name} ({self.data_year})'
 
+    def courses(self):
+        courses = []
+
+        # get the coursegroups and sort by course type
+        types = [CourseType.MUST, CourseType.FROM_LIST, CourseType.CHOICE,
+                 CourseType.CORNER_STONE, CourseType.SUPPLEMENTARY]
```

Comment:
Use `ALL_COURSE_TYPES` from `.models`, a single source of truth is preferred over redefining this list

Reply:
changed